package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "maas-schemas",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
I'd say it should be a minor bump because `ticketCollectionPoint` is a newly introduced property